src/style.css

```diff
@@ -1 +1,45 @@
 /* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--1 :first-child {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
The path to the images is not consistent. Here you're using `./images/star-active.svg`, while in other places you're using `images/star-active.svg`. It's not an error, but it's better to keep the code consistent.